src/Maestro/maestro-angular/src/app/page/channel-graph/channel-graph.component.ts

```diff
@@ -0,0 +1,136 @@
+import { Component, Input, OnChanges, SimpleChanges, AfterContentInit } from '@angular/core';
+import { graphlib, render } from 'dagre-d3';
+import { select } from 'd3';
+
+import { FlowGraph, FlowRef, FlowEdge } from 'src/maestro-client/models';
+import { RepoNamePipe } from 'src/app/pipes/repo-name.pipe';
+import { analyzeAndValidateNgModules } from '@angular/compiler';
+
+function getRepositoryShortName(repo:string): string {
+  return `${RepoNamePipe.prototype.transform(repo)}`;
+}
+function getNodeLabel(node:FlowRef): string {
+  let label = `${getRepositoryShortName(node.repository)}\n`;
+  label = `${label}${node.branch}\n`;
```

Comment:
>label = `${label}${node.branch}\n` [](start = 2, length = 34)

Why not just put this in the first line? Took me a while to realize the first value was "label" again, so this wasn't throwing out the original value. :-)